snowflake/assets/configuration/spec.yaml

```diff
@@ -67,6 +67,17 @@ files:
         value:
           type: string
           example: <PASSWORD>
+      - name: role
+        enable: true
```

Comment:
Is this supposed to be `enabled`? From docs: https://datadoghq.dev/integrations-core/meta/config-specs/#example-file-consumer

Reply:
removing attribute since it's marked as requied.